config/operator/all-in-one/cluster_role.template.yaml

```diff
@@ -6,6 +6,12 @@ kind: ClusterRole
 metadata:
   name: elastic-operator
 rules:
+- apiGroups:
```

Comment:
Only the cluster role of the all-in-one manifest is updated since this feature only makes sense when the operator is watching more than one namespace.